datastore/protobuf/datastore.proto

```diff
@@ -71,47 +72,58 @@ message Link {
 }
 
 message TSMetadata {
-  string version = 1;
-  string type = 2;
-  string title = 3;
-  string summary = 4;
-  string keywords = 5;
-  string keywords_vocabulary = 6 [json_name = "keywords_vocabulary"];
-  string license = 7;
-  string conventions = 8;
-  string naming_authority = 9 [json_name = "naming_authority"];
-  string creator_type = 10 [json_name = "creator_type"];
-  string creator_name = 11 [json_name = "creator_name"];
-  string creator_email = 12 [json_name = "creator_email"];
-  string creator_url = 13 [json_name = "creator_url"];
-  string institution = 14;
-  string project = 15;
-  string source = 16;
-  string platform = 17;
-  string platform_vocabulary = 18 [json_name = "platform_vocabulary"];
-  string standard_name = 19 [json_name = "standard_name"];
-  string unit = 20;
-  string instrument = 21;
-  string instrument_vocabulary = 22 [json_name = "instrument_vocabulary"];
-  repeated Link links = 23;
+  // --- BEGIN non-string metadata -----------------
+  repeated Link links = 1;
+  // --- END non-string metadata -----------------
+
+  // --- BEGIN string metadata (handleable with reflection) -----------------
+  string version = 2;
+  string type = 3;
+  string title = 4;
+  string summary = 5;
+  string keywords = 6;
+  string keywords_vocabulary = 7 [json_name = "keywords_vocabulary"];
+  string license = 8;
+  string conventions = 9;
+  string naming_authority = 10 [json_name = "naming_authority"];
+  string creator_type = 101 [json_name = "creator_type"];
+  string creator_name = 12 [json_name = "creator_name"];
+  string creator_email = 13 [json_name = "creator_email"];
+  string creator_url = 14 [json_name = "creator_url"];
+  string institution = 15;
+  string project = 16;
+  string source = 17;
+  string platform = 18;
+  string platform_vocabulary = 19 [json_name = "platform_vocabulary"];
+  string standard_name = 20 [json_name = "standard_name"];
+  string unit = 21;
+  string instrument = 22;
+  string instrument_vocabulary = 23 [json_name = "instrument_vocabulary"];
```

Comment:
This is a breaking schema change... which is probably fine, but we should let the ingestion team know.

Reply:
Good point! Will do!